Source/GR32_Transforms.pas

```diff
@@ -1787,9 +1787,8 @@ procedure TRadialDistortionTransformation.PrepareTransform;
     Exit;
 
   // center / focal point relative to which all (un)distortions are calculated
-  FFocalPoint := TPointF.Create(
-    (SrcRect.Right + SrcRect.Left) / 2,
-    (SrcRect.Bottom + SrcRect.Top) / 2);
+  FFocalPoint.x := (SrcRect.Right + SrcRect.Left) / 2;
+  FFocalPoint.y := (SrcRect.Right + SrcRect.Left) / 2;

```

Comment:
Isn't it obvious?

Reply:
oh.... yes, of course. Too exited about it worked...